### ISSUE TITLE
AloeStackViewController to support child UIViewControllers as a row

### ISSUE THREAD

**Issue by richardtop** [airbnb/AloeStackView#53]

**Comment by MikeGlotov** [comment]
H @richardtop, I'm not the maintainer but I was able to use VCs as Aloe's children. You could try this in your VC:
```
    let vc = ...someVC
    addChild(section)
    vc.didMove(toParent: self)
    stackView.addRow(vc.view)
```
And don't forget abot row insets. Hope this would be helpful.

**Comment by richardtop** [comment]
Yes, but that's not the 1st class support. The issue is about making AloeStackViewController to support child ViewControllers natively (i.e. it would do the manual work you've referenced)

**Comment by MikeGlotov** [comment]
> Hey @richardtop thank you for the suggestion, this sounds like a great idea, we'll look into adding support for this via `AloeStackViewController` in the future.

Do you guys want to maintain the same interface or extend it?
